editor/animation_track_editor.h

```diff
@@ -600,8 +609,24 @@ class AnimationTrackEditor : public VBoxContainer {
 		Vector<Key> keys;
 	};
 
+	struct KeyClipboard {
+		int top_track;
+
+		struct Key {
+			Animation::TrackType track_type;
+			int track;
+			float time = 0;
+			float transition = 0;
+			Variant value;
+		};
+		Vector<Key> keys;
+	};
+
 	Vector<TrackClipboard> track_clipboard;
+	KeyClipboard key_clipboard;
 
+	void _set_key_clipboard(int top_track, float top_time, RBMap<SelectedKey, KeyInfo> keymap);
```

Comment:
```suggestion
	void _set_key_clipboard(int p_top_track, float p_top_time, RBMap<SelectedKey, KeyInfo> &p_keymap);
```
Pass by reference or you'll copy the whole map